virttest/cartesian_config.py

```diff
@@ -606,15 +606,15 @@ def _substitution(value, d):
     if "$" in value:
         start = 0
         st = ""
+        d_flat = _drop_suffixes(d)
```

Comment:
Good catch, just suggest splitting this into another commit, thanks.

Reply:
Indeed, and this is performance critical location, great!